templates/main.cf.j2

```diff
@@ -896,8 +896,11 @@ bounce_queue_lifetime = {{ postfix_bounce_queue_lifetime }}
 {% endif %}
 
 # https://stafwag.github.io/blog/blog/2018/03/04/postfix-smarthost-with-authentication/
-smtp_use_tls = {{ postfix_smtp_use_tls | ternary('yes', 'no') }}
```

Comment:
With Postfix 2.3+, `smtp_tls_security_level` should be used instead (https://www.postfix.org/postconf.5.html#smtp_use_tls), which overrides `smtp_use_tls` (https://www.postfix.org/postconf.5.html#smtp_tls_security_level).